### DISCUSSION TITLE
毕业十年 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#17] [Announcements]
# 毕业十年 - forecho's Blog

引言
又是一年一度的高考月，也是一年一度的毕业季。与往年不同的是今年是我大学毕业 10 年了。所以本月我打算写一篇文章算是纪念一下吧。
高考
13 年前（2009 年）的 6 月我参加完高考，考试结束那晚我们班还组织了去网吧通宵打游戏的活动。然后大家就各自散了。
再见面就是去学校填写志愿的时候了，当时我的高考成绩具体是多少分来着，我也记不清了，本科没考上，只能上大专，反正预期也不高，上大专也行吧。当时上的高中学校也不是很好，班上考上本科的人数屈指可数。
到了填志愿的时候，我是先选的学校再选的专业，当时就对计算机感兴趣于是找了一个自己觉得还不错的学校。
记得当时很多同学都在为填志愿发愁，因为中国的学生大部分的首要任务就是学习，不停的学习，除了学习，他们甚至都不知道自己的兴趣是什么？一到高考填志愿的时候就犯愁。
过来人的建议就是先选学校，志愿入学之后还可以再转。选专业一定选自己感兴趣的专业，不要因为目前行业工资高而选，因为等你毕业之后那个行业什么情况可不好说。

https://blog.forecho.com/ten-years-after-graduation.html

**Comment by qianyugang** [comment]
当时在武汉面的几个公司都忘记了，我记得有个弄什么垃圾视频复制的公司我俩都去了的。
